src/app/openmrs-api/oder-set-resource.service.spec.ts

```diff
@@ -0,0 +1,15 @@
+// import { TestBed, inject } from '@angular/core/testing';
```

Comment:
Why do we have this file without any tests? Also check the typo in the file name.